### ISSUE TITLE
Adobe/Omniture POST method tracking shows duplicate call in panel

### ISSUE THREAD

**Issue by ekallevig** [sean-adams/dataslayer#50]
When the Adobe tracking payload reaches a certain size, Adobe's AppMeasurement.js will switch the request method from an image/pixel GET request with query string parameters to an XHR POST request with the parameters passed in the body of the POST. This is presumably due to URL length limitations.

In the POST cases, the browser makes some kind of an "internal" request that returns a 400 before proceeding to make the regular "network" request (which returns a 200). Only the regular request appears in the network tab.

Both of these requests however are being passed to dataSlayer's request listener (https://github.com/sean-adams/dataslayer/blob/master/src/Panel/dsPanel.js#L950), and the result is dataSlayer displaying the single tracking call as two separate, identical calls.

When adding a breakpoint to the extension's request listener, I was able to see that the "internal" request's "response" object looks like this:
```  "response": {
    "status": 400,
    "statusText": "Service Worker Fallback Required",
    "httpVersion": "HTTP/1.1",
    "headers": [],
    "cookies": [],
    "content": {
      "size": 0,
      "mimeType": "text/plain"
    },
    "redirectURL": "",
    "headersSize": 49,
    "bodySize": 0,
    "_transferSize": 0
  },
```

I would think either the 400 response status or the statusText could be keyed off to discard these events and avoid the duplication issue.

**Comment by ekallevig** [comment]
Also worth noting this same issue occurs in Adobe's own new Chrome debugger extension (https://chrome.google.com/webstore/detail/adobe-experience-cloud-de/ocdmogmohccmeicdhlhhgepeaijenapj/related). I'm guessing the POST method is not commonly triggered and thus was overlooked.

**Comment by sean-adams** [comment]
This issue should be fixed in 1.0 (releasing soon).
